### ISSUE TITLE
Business key should be a required field

### ISSUE THREAD

**Issue by kahowell** [esbtools/esb-message-admin#59]
This will allow the field to be available as a column for sorting in the error tab grid, etc.

**Comment by derek63** [comment]
To clarify...this should be a required field when persisting an error message via the REST endpoint?

**Comment by kahowell** [comment]
Yeah, this is one of the columns we've identified for the error tab grid, and thus it should be a required field.
